session.go

```diff
@@ -374,58 +374,61 @@ func map2Nmap(m map[string]interface{}) (*nebula.NMap, error) {
 // construct go-type to nebula.Value
 func value2Nvalue(any interface{}) (value *nebula.Value, err error) {
```

Comment:
prefer do not use `any`, because it's a keyword now...
it's involved a long time ago, maybe we could change it in this pr?

Reply:
make sense will use a decent name later, how about param?

Reply:
On a flight being taking off, will do it after landing.

Reply:
@HarrisChu it's done :)